src/polyhash/index.js

```diff
@@ -360,7 +354,7 @@ function toCluster(inputPolygon, locationIdPrecision) {
         // Then convert intersection polygon points into locationIdes
         intersectionLocationIdsCells = []
         turfMeta.coordEach(_newTestPolygon, function (currentCoord) {
-          const locationId = unl.encode(currentCoord[0], currentCoord[1], locationIdPrecision)
+          const locationId = unl.encode(currentCoord[1], currentCoord[0], locationIdPrecision)
```

Comment:
Do we need to make changes somewhere else?

Reply:
yes, the toPolyhash function is now inconsistent with toCluster, making the change.